semana14/aula5/F4System/src/mobileMission.ts

```diff
@@ -0,0 +1,20 @@
+import {f4JSONFileManager, JSONFileManager} from "./jsonFileManager";
+import {Teacher} from "./teacher";
+import {Student} from "./student";
+import {Mission} from "./mission";
+
+export class MobileMission extends  Mission {
+    constructor(public classNumber: number, startDate: string, endDate: string, teachers: Teacher[], students: Student[]) {
+    super(startDate, endDate, teachers, students);
+    this.classNumber = classNumber;
+    this.startDate = startDate;
+    this.endDate = endDate;
+    this.teachers = [];
+    this.students = [];
+  }
+
+  createClass(classNumber: number, startDate: string, endDate: string, teachers: Teacher[], students: Student[]): void {
```

Comment:
Se você já tiver todas essas informações na classe, você não precisa pedir de novo! É só usar o `this.startDate, this.endDate` e assim vai